controllers/articles.controller.js

```diff
@@ -0,0 +1,17 @@
+const { fetchArticleById } = require("../models/articles.model");
+
+exports.getArticleById = (req, res, next) => {
+  const { article_id } = req.params;
+
+  if (isNaN(article_id)) {
+    return res.status(400).send({ msg: "Invalid data type" });
+  }
+
```

Comment:
This condition is not necessary to include. If a malformed article_id goes into the query, psql will throw an error by default.